velox/exec/tests/TableWriteTest.cpp

```diff
@@ -475,13 +456,13 @@ class TableWriteTest : public HiveConnectorTestBase {
       const std::string& targetDir) {
     verifyPartitionedDirPath(filePath, targetDir);
     if (commitStrategy_ == CommitStrategy::kNoCommit) {
-      ASSERT_TRUE(RE2::FullMatch(
-          filePath.filename().string(), "0[0-9]+_0_test_cursor.+"))
+      // NOTE: the query id is empty in unit test.
```

Comment:
Can we make it non-empty?

Reply:
Good point. I thought about and add a query id in task cursor.